src/System.Runtime/tests/System/ActivatorTest.netstandard1.7.cs

```diff
@@ -0,0 +1,121 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+using System;
+using System.Globalization;
+using System.Reflection;
+using System.Runtime.InteropServices;
+using Xunit;
+
+namespace System.Tests
+{
+
+    public class ActivatorTestsExtended
+    {
+        class ClassWithPrivateCtor
+        {
+            static ClassWithPrivateCtor() { Flag.Reset(100); }
+            private ClassWithPrivateCtor() { Flag.Increase(200); }
+            public ClassWithPrivateCtor(int i) { Flag.Increase(300); }
+        }
+        class ClassWithPrivateCtor2
+        {
+            static ClassWithPrivateCtor2() { Flag.Reset(100); }
+            private ClassWithPrivateCtor2() { Flag.Increase(200); }
+            public ClassWithPrivateCtor2(int i) { Flag.Increase(300); }
+        }
+        class ClassWithPrivateCtor3
+        {
+            static ClassWithPrivateCtor3() { Flag.Reset(100); }
+            private ClassWithPrivateCtor3() { Flag.Increase(200); }
+            public ClassWithPrivateCtor3(int i) { Flag.Increase(300); }
+        }
+
+
+        public class IsTestedAttribute : Attribute
+        {
+            private bool flag;
+            public IsTestedAttribute(bool flag)
+            {
+                this.flag = flag;
+
+            }
+        }
+        [IsTestedAttribute(false)]
+        class ClassWithIsTestedAttribute { }
+        [Serializable]
+        class ClassWithSerializableAttribute { }
+        [IsTestedAttribute(false)]
+        class MBRWithIsTestedAttribute : MarshalByRefObject { }
+        class Flag
+        {
+            public static int cnt = 0;
```

Comment:
It's ok as long as it's only used by a single test, or worst by tests in the same class.  By default we parallelize xunit tests at the class level, such that tests from the same class don't run in parallel, but tests from different classes may.  There's also no guarantee on the ordering the tests will run in, even within the same test class.